Artifacts/windows-azuredevops-multiple-agents/run.ps1

```diff
@@ -0,0 +1,122 @@
+# Downloads the Azure DevOps Pipelines Agent and installs specified instances on the new machine
+# under C:\agents\ and registers with the Azure DevOps Pipelines agent pool
+
+# Enable -Verbose option
+[CmdletBinding()]
+Param
+(
+	[Parameter(Mandatory=$true)]
+	[string]$Account,
+
+	[Parameter(Mandatory=$true)]
+	[String]$PersonalAccessToken,
+
+	[string]$AgentName = $env:COMPUTERNAME,
+
+	[string]$AgentNamePrefix= "dtl",
```

Comment:
Remove default, set it in the `artifactfile.json` instead, so it will also show in the UI to the user.